src/Store/SessionStore.php

```diff
@@ -38,25 +38,34 @@ class SessionStore implements StoreInterface
     protected $state = [];
 
     /**
-     * Create a store from the given request getting any initial state from the session of the request
+     * @param Member $member
+     */
+    public function __construct(Member $member = null)
+    {
+        $this->member = $member;
```

Comment:
How about `setMember($member)` for free code coverage

Reply:
I think it was made to explicitly short circuit `setMember` because we don't want the method to be cleared. In retrospect the method is already clear so it's not a concern 🤔 .

Good idea! 👍